assets/scss/_nav.scss

```diff
@@ -116,7 +116,128 @@
                 overflow-x: auto;
                 white-space: nowrap;
                 -webkit-overflow-scrolling: touch;
+
             }
         }
     }
-}
\ No newline at end of file
+}
+
+// Icons
+#main_navbar {
+    li i {
+    padding-right: 0.5em;
+
+        &:before {
+            display: inline-block;
+            text-align: center;
+            min-width: 1em;
+        }
+    }
+  .alert {
+    background-color: inherit;
+    padding:0;
+    color: $secondary;
+    border: 0;
+    font-weight: inherit;
+
+        &:before {
+            display: inline-block;
+            font-style: normal;
+            font-variant: normal;
+            text-rendering: auto;
+            -webkit-font-smoothing: antialiased;
+            font-family: "Font Awesome 5 Free";
+            font-weight: 900;
+            content: "\f0d9";
+            padding-left: 0.5em;
+            padding-right: 0.5em;
+        }
+    }
+}
+
+// Foldable sidebar menu
+nav.foldable-nav {
+
+    &#td-section-nav {
+        position: relative;
+    }
+
+    &#td-section-nav label {
+        margin-bottom: 0;
+        width: 100%;
+    }
+
+    .td-sidebar-nav__section, .with-child ul {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+    }
+
+    .ul-1 > li {
+        padding-left: 1.5em;
+    }
+
+    ul.foldable {
+        max-height: 0;
+        overflow: hidden;
+        transition: max-height 0.5s cubic-bezier(0, 1, 0, 1);
+    }
+
+    input:checked ~ ul.foldable {
+        max-height: 100000vmax;
+        transition: max-height 1s ease-in-out;
+    }
+
+    input[type=checkbox] { display: none; }
+
+    .with-child, .without-child {
+        position: relative;
+        padding-left: 1.5em;
+    }
+
+    .ul-1 .with-child > label:before {
+        display: inline-block;
+        font-style: normal;
+        font-variant: normal;
+        text-rendering: auto;
+        -webkit-font-smoothing: antialiased;
+        font-family: "Font Awesome 5 Free"; font-weight: 900; content: "\f0da";
+        position: absolute;
+        left: 0.1em;
+        padding-left: 0.4em;
+        padding-right: 0.4em;
+        font-size: 1em;
+        color: $gray-900;
+        transition: all 0.5s;
+        &:hover{
+        transform: rotate(90deg);
+        }
+    }
+
+    .ul-1 .with-child > input:checked ~ label:before {
+        color: $primary;
+        transform: rotate(90deg);
+        transition: transform 0.5s;
+    }
+
+    .with-child ul { margin-top: 0.1em; }
+
+}
+
+@media (hover: hover) and (pointer: fine) {
+
+    nav.foldable-nav {
+
+        .ul-1 .with-child > label:hover:before {
+            color: $primary;
+            transform: rotate(30deg);
```

Comment:
I think this effect needs to be removed, it is not needed
**transform**
On hover, you can already see what changes color

Reply:
This change actually was not from me, it was from the docsy update (needed because we were fully overwriting the docsy `_nav.scss` file).

I have now moved these overrides into `_styles_project.scss` (where docsy actually recommends all custom staying is put).

Reply:
I guess the question is how about we override the default behavior, to remove `transform` animation in Kubeflow website? Because the default behavior is to rotate the pointer when being hovered over, which is a bit strange. (If the sub-section is already expanded, the pointer will rotate backward by 30 degree)

Reply:
@zijianjoy take that up on the [`google/docsy` repo](https://github.com/google/docsy) if you want, but its unrelated to this PR.

Here is [the exact upstream docsy code](https://github.com/google/docsy/blob/3ae94a0573a8c14f61c832b6e7fdcc46af62187c/assets/scss/_nav.scss#L206-L210).